docs/content/guide/component.ngdoc

```diff
@@ -252,8 +255,8 @@ it upwards to the heroList component, which updates it.
     templateUrl: 'editableField.html',
     controller: EditableFieldController,
     bindings: {
-      fieldValue: '@',
-      fieldType: '@',
+      fieldValue: '<',
+      fieldType: '<?',
```

Comment:
In this specific case. Otherwise the number of watches is the same, no (1 for interpolation, 1 for < binding)